.github/workflows/experimentalJUnitVersion.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Experimental JUnit5 build
+
+on:
+  push:
+    branches: 'master'
+    tags-ignore:
+      - 'v**'
+      - 'releaseTrigger'
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        junit5: [ '5.0', '5.1', '5.2', '6.0', '6.1', '6.2' ]
+    name: with 5.${{ matrix.junit5 }}
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
Why 14? Default-version if Jupiter and Pioneer is 8!

Reply:
the target version is 8 - does not mean you can not build it with newer versions, that is just a minor detail, but the latest LTS is 14 so i would go with our experimental topics with a recent version

Reply:
Latest LTS is 11, next will be 17 :)

14 is just current supported version until September (?) when 15 comes out :)

Reply:
wait isnt it every third release an lts... did i miss again something?

Reply:
my bad... i somehow had in my mind that every 3rd release is an LTS (not sure where this is coming from) - but than 11 would make more sense, in my opinion

Reply:
I had the same misinformation in my mind for a long time too, but it's every 3rd year (= sixth release), not every 3rd release :)